### ISSUE TITLE
JVM Standalone launcher does not work

### ISSUE THREAD

**Issue by tpasternak** [VirtusLab/scala-cli#542]
On 7d3144d

Steps:
1. `./mill -i cli.standaloneLauncher`
2. `./out/cli/standaloneLauncher/dest/launcher package --standalone examples/scala-files -o scalaFiles2`

Expected:
Package is built

Actual:
```
Exception in thread "main" java.lang.NoSuchMethodError: 'java.io.File coursier.paths.CachePath.defaultArchiveCacheDirectory()'
	at coursier.cache.CacheDefaults$.archiveCacheLocation$lzycompute(CacheDefaults.scala:18)
	at coursier.cache.CacheDefaults$.archiveCacheLocation(CacheDefaults.scala:18)
	at coursier.cache.ArchiveCache$.apply(ArchiveCache.scala:141)
	at coursier.jvm.JvmCache$.apply(JvmCache.scala:26)
	at scala.build.options.BuildOptions.javaHomeManager$lzycompute(BuildOptions.scala:240)
	at scala.build.options.BuildOptions.javaHomeManager(BuildOptions.scala:237)
	at scala.cli.commands.SharedOptions.bloopRifleConfig(SharedOptions.scala:189)
	at scala.cli.commands.Package$.run(Package.scala:45)
	at scala.cli.commands.Package$.run(Package.scala:31)
	at caseapp.core.app.CaseApp.main(CaseApp.scala:149)
	at caseapp.core.app.CommandsEntryPoint.main(CommandsEntryPoint.scala:116)
	at scala.cli.ScalaCli$.main0(ScalaCli.scala:173)
	at scala.cli.ScalaCli$.main(ScalaCli.scala:114)
	at scala.cli.ScalaCli.main(ScalaCli.scala)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at coursier.bootstrap.launcher.Bootstrap.main(Bootstrap.java:109)
	at coursier.bootstrap.launcher.ResourcesLauncher.main(ResourcesLauncher.java:79)
```
https://github.com/VirtusLab/scala-cli/runs/4804289995?check_suite_focus=true#step:7:648

**Comment by tpasternak** [comment]
Seems that there's a version conflict between bootstrap code that coursier includes each time

https://github.com/coursier/coursier/blob/bc9eeb9d0ffb46ba6627ed5a8b162f7906fb2899/modules/launcher/src/main/scala/coursier/launcher/BootstrapGenerator.scala#L218-L229

and the version of coursier that `scala-cli`depends on

**Comment by tpasternak** [comment]
closed in #544
